src/style.css

```diff
@@ -1 +1,93 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+  font-style: normal;
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url("https://fonts.googleapis.com/css2family=Roboto:wght@300;700&display=swap");
+  font-style: bold;
+  font-weight: 700;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+  --main-color: #00acdc;
+}
+
+.container {
+  padding: 0 50px;
+
+  display: flex;
+  align-items: center;
+
+  justify-content: space-between;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  position: relative;
+  width: 40px;
+  height: 40px;
+  background-image: url("./images/logo.png");
+  background-repeat: no-repeat;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  font-size: 12px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  text-transform: uppercase;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav__list:hover:after {
+  background: red;
+  transition: ease-out 3s;
+}
+
+.nav__list:not(:last-child) {
+  margin-right: 20px;
+}
+
+.box {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  padding: 22.5px 0;
+
+  color: black;
+  text-decoration: none;
+}
+
+.box:hover {
+  text-decoration: none;
+  color: var(--main-color);
+}
+
+.is-active {
+  color: var(--main-color);
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  background: var(--main-color);
+  height: 4px;
+  width: 37px;
```

Comment:
don't hardcode the width. it should work properly if we set `is-active` to any other link